server/src/test/java/com/microsoft/java/bs/core/internal/server/BuildTargetServerIntegrationTest.java

```diff
@@ -370,6 +467,376 @@ void testCompilingSingleProjectServer() {
     });
   }
 
+  @Test
+  void testRunTestsProjectServer() {
```

Comment:
Maybe we can consider to separate the test feature tests to a separate file since this is a big one. And I have a feeling that more and more cases will be added for different test framework, and different usage for the same framework.

Reply:
Happy to move the testing to a different integ test.